eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -35,6 +35,10 @@ jobs:
     pool: ${{ parameters.pool }}
     condition: ${{ parameters.condition }}
 
+    # Only run Rich Nav indexer on Windows NT_x64 platform for now
+    ${{ if and(eq(parameters.buildConfig, 'checked'), eq(parameters.platform, 'Windows_NT_x64')) }}:
+      enableRichCodeNavigation: true
```

Comment:
Instead of putting this here flow the parameter as a jobParameter in the entrypoint yml, `rich-nav.yml`

Reply:
Good call! I changed that as well